cypress/fixtures/visitor_can_register.json

```diff
@@ -0,0 +1,14 @@
+{
+  "success": true,
```

Comment:
please double check that this is really part of the response and received as a boolean

Reply:
Have added logic to convert the response to boolean and have updated the fixture to reflect the actual response